### DISCUSSION TITLE
Is incompability with Sonoma fixed?

### DISCUSSION THREAD

**Discussion by idoan** [ChefKissInc/NootedRed#277] [Q&A] [answered]
Hi, I have recently inatalled Ventura on AMD laptop . While I am considering to update it to Sonoma, I have seen the issue #235 which warns about the occasional crashes due to incompability. Is this fixed by now? Can I update my ventura to Sonoma without any nootedred related issues? 

Thanks.

**Comment by VisualEhrmanntraut** [comment, marked as answer]
That issue will be fixed in the very near future.
